Modules/FindNetCDF.cmake

```diff
@@ -230,6 +242,10 @@ foreach( _comp IN LISTS _search_components )
         INTERFACE_LINK_LIBRARIES ${NetCDF_${_comp}_LIBRARIES} )
       if( NOT _comp MATCHES "^(C)$" )
         target_link_libraries(NetCDF::NetCDF_${_comp} INTERFACE NetCDF::NetCDF_C)
+      else()
+	if(NetCDF_PARALLEL)
+	  target_link_libraries(NetCDF::NetCDF_${_comp} INTERFACE MPI::MPI_C)
+	endif()
```

Comment:
`_comp` is `C` in this else block.  Making it explicit doesn't change the result, just makes reading easy-er.
Is an indent missing? Or is this a tab issue?
Otherwise looks good.

Reply:
I made it explicit.

I don't know why it shows like that. See how it looks from here

https://github.com/kgerheiser/CMakeModules/blob/b636f83f5a1a9e98417c162b6ce60291d4d11488/Modules/FindNetCDF.cmake#L248

Reply:
When you checkout your branch, the indentation is bonkers. See attached screen shot.
Also, get rid of trailing blanks on Line 181.
![Screen Shot 2020-09-02 at 12 51 59 PM](https://user-images.githubusercontent.com/11394126/92012936-19255b80-ed1b-11ea-8cc7-b1f7ffe7379f.png)

You are using tabs and I think the tab-expansion in your editor is turned off or messed up.